e2e/tests/caseManagementOrder_test.js

```diff
@@ -68,7 +69,44 @@ Scenario('local authority creates CMO', async (I, caseViewPage, draftCaseManagem
   await draftCaseManagementOrderEventPage.enterRecital('Recital 1', 'Recital 1 description');
   I.click('Continue');
   await draftCaseManagementOrderEventPage.enterSchedule(schedule);
+  I.click('Continue');
+  draftCaseManagementOrderEventPage.markToReviewedBySelf();
+  await I.completeEvent('Submit');
+  assertCanSeeDraftCMO(I, caseViewPage, draftCaseManagementOrderEventPage.staticFields.statusRadioGroup.selfReview);
+  await caseViewPage.goToNewActions(config.applicationActions.draftCaseManagementOrder);
+  await draftCaseManagementOrderEventPage.validatePreviousSelectedHearingDate('1 Jan 2050');
+});
+
+// This scenario relies on running after 'local authority creates CMO'
+Scenario('Other parties cannot see the draft CMO when it is marked for self review', async (I, caseViewPage, draftCaseManagementOrderEventPage) => {
+  // Ensure the selection is self review
+  await caseViewPage.goToNewActions(config.applicationActions.draftCaseManagementOrder);
+  skipToReview(I);
+  draftCaseManagementOrderEventPage.markToReviewedBySelf();
+  await I.completeEvent('Submit');
+  assertCanSeeDraftCMO(I, caseViewPage, draftCaseManagementOrderEventPage.staticFields.statusRadioGroup.selfReview);
+
+  for (let userDetails of allOtherPartyDetails) {
+    await assertUserCannotSeeDraftOrders(I, userDetails);
+  }
+});
+
+// This scenario relies on running after 'local authority creates CMO'
+// Currently send to judge does the same as party review
+Scenario('Other parties can see the draft CMO when it is marked for party review', async (I, caseViewPage, draftCaseManagementOrderEventPage) => {
+  // Ensure the selection is party review
+  await caseViewPage.goToNewActions(config.applicationActions.draftCaseManagementOrder);
+  skipToReview(I);
```

Comment:
We needs await here right? I think I saw them in another PR. So merging other into that would fix it.